File: app/src/main/java/andiag/coru/es/welegends/rest/entities/persist/Champion.java
package andiag.coru.es.welegends.rest.entities.persist;

import java.util.HashMap;
import java.util.Map;

/**
 * Created by andyqm on 06/06/2016.
 */
public class Champion {
    private String version;
    private String id;
    private String key;
    private String name;
    private String blurb;
    private Image image;
    private String partype;
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    public String getVersion() {
        return version;
    }

    public void setVersion(String version) {
        this.version = version;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBlurb() {
        return blurb;
    }

    public void setBlurb(String blurb) {
        this.blurb = blurb;
    }

    public Image getImage() {
        return image;
    }

    public void setImage(Image image) {
        this.image = image;
    }

    public String getPartype() {
        return partype;
    }

    public void setPartype(String partype) {
        this.partype = partype;
    }

    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }
}

File: WeLegends/app/src/main/java/es/coru/andiag/welegends/models/entities/Champion.kt
package es.coru.andiag.welegends.models.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.orm.SugarRecord
import java.io.Serializable

/**
 * Created by Canalejas on 08/12/2016.
 */

class Champion : SugarRecord(), Serializable {

    @SerializedName("id") @Expose var name: String? = null
    @Expose var version: String? = null
    @Expose var key: String? = null
    @Expose var title: String? = null
    @Expose var blurb: String? = null
    @Expose var image: Image? = null
    @Expose var partype: String? = null

}
